### DISCUSSION TITLE
Running or Stopped

### DISCUSSION THREAD

**Discussion by UcraNerd** [d4rken-org/sdmaid-se#358] [Ideas]
**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is.

> Nope

**Describe the solution you'd like**

> I would like it if SD Maid could add an application tag for the running or disabled state, like the system or disabled one

**Describe alternatives you've considered**

> Maybe adding a circle to the app icon: red if stopped, green if currently running

**Describe why this would be in the interest of all users**
Describe why this is not a niche feature, but would be a benefit for the majority of users.

> It's usefull if you want to force stop the application that you're not currently using for lighting the memory and battery usage

**Additional context**

**Comment by d4rken** [comment]
I'm not sure this is within scope. I regret SD Maid 1 focusing a bit on this in earlier iterations. Whether an app is "running" or not continues to become more difficult to judge with every new Android version due to various different states an app can be in that are theoretically all "running" states.
